shell/gpu/gpu_surface_metal.mm

```diff
@@ -152,7 +152,7 @@
     return true;
   };
 
-  return std::make_unique<SurfaceFrame>(std::move(surface), submit_callback);
+  return std::make_unique<SurfaceFrame>(std::move(surface), true, submit_callback);
```

Comment:
Need to verify that we are using the right flags with the metal surface creation to support readback. Finding the optimal settings for Metal will be work for another PR, but we need to make sure the boolean used here is correct for now.

Reply:
Confirmed. Line 28 in the constructor in this same file specifically sets the "framebuffer only" flag on the CAMetalLayer to NO:

```
26:  // Flutter needs to read from the color attachment in cases where there are effects such as
27:  // backdrop filters.
28:  layer.get().framebufferOnly = NO;
```